network-api/networkapi/templates/pages/base.html

```diff
@@ -28,8 +28,8 @@
   </head>
   <body class="{% block bodyclass %}{% endblock %}" id="view-{% block bodyID %}{% endblock %}">
     {% if user.is_authenticated %}
-    <div class="authenticated-page-edit" style="position: fixed; bottom:0; background: teal; padding: 0.25em; z-index:1000">
-      <a style="color: white" href="/cms/pages/{{page.id}}/edit/">Edit</a>
+    <div class="authenticated-page-edit">
```

Comment:
we don't need to keep this outer `div`, wagtail already puts the user bar code in its own single div.